lib/Entity/Media.php

```diff
@@ -834,23 +834,21 @@ private function assessDimensions(): void
 
     /**
      * Release an image from image processing
-     * @param $md5
-     * @param $fileSize
+     * @param $img
      */
-    public function release($md5, $fileSize, $height, $width)
+    public function release($img)
```

Comment:
Sorry if my comments have confused you. It is better to have explicit parameters as you had it before (you could even add types to them if you wanted).

Reply:
Thanks for this :) I've added a new commit [here ](https://github.com/xibosignage/xibo-cms/pull/2602/commits/542d406e34ee2b22954a661dc8cb7a74531a672e). Requesting review again, please. Thank you!